I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Alinor.rpy", line 26, in script
    $ buttons('trading_area', 4, ['alinor_capital_gorod' , 'alinor_capital_ul_dom' , 'alinor_capital_pereulok', 'alinor_lavka_torgovca', 'vokzal'])
  File "game/Alinor.rpy", line 26, in <module>
    $ buttons('trading_area', 4, ['alinor_capital_gorod' , 'alinor_capital_ul_dom' , 'alinor_capital_pereulok', 'alinor_lavka_torgovca', 'vokzal'])
  File "game/System.rpy", line 46, in buttons
    renpy.call_screen('location_buttons')
  File "renpy/common/00action_data.rpy", line 50, in get_selected
    return getattr(self.object, self.field) == self.value
AttributeError: 'StoreModule' object has no attribute ''

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Alinor.rpy", line 26, in script
    $ buttons('trading_area', 4, ['alinor_capital_gorod' , 'alinor_capital_ul_dom' , 'alinor_capital_pereulok', 'alinor_lavka_torgovca', 'vokzal'])
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\ast.py", line 848, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\python.py", line 1812, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/Alinor.rpy", line 26, in <module>
    $ buttons('trading_area', 4, ['alinor_capital_gorod' , 'alinor_capital_ul_dom' , 'alinor_capital_pereulok', 'alinor_lavka_torgovca', 'vokzal'])
  File "game/System.rpy", line 46, in buttons
    renpy.call_screen('location_buttons')
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\exports.py", line 2655, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\ui.py", line 287, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 2624, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 2999, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 511, in visit_all
    d.visit_all(callback)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 511, in visit_all
    d.visit_all(callback)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 511, in visit_all
    d.visit_all(callback)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\screen.py", line 405, in visit_all
    self.child.visit_all(callback)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 511, in visit_all
    d.visit_all(callback)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 513, in visit_all
    callback(self)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\core.py", line 2999, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\behavior.py", line 796, in per_interact
    if self.is_selected():
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\behavior.py", line 786, in is_selected
    return is_selected(self.action)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\behavior.py", line 356, in is_selected
    return any(is_selected(i) for i in action)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\behavior.py", line 356, in <genexpr>
    return any(is_selected(i) for i in action)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\display\behavior.py", line 359, in is_selected
    return action.get_selected()
  File "renpy/common/00action_data.rpy", line 50, in get_selected
    return getattr(self.object, self.field) == self.value
AttributeError: 'StoreModule' object has no attribute ''

Windows-7-6.1.7601-SP1
Ren'Py 6.99.14.1.3218
 
Sat Jul 20 08:39:55 2019
